content/design/line-styles/index.html

```diff
@@ -2,65 +2,136 @@
-    code: '@include bb-border(dark)'
-    example: '<div class="bb-subsection-heading" style="border-bottom: 1px solid #ccc;">Dark separator</div>'
-    styling: '1px solid #ccc'
-
-itemsIsolated:
-  - usage: This line style highlights the active navigation item in the <a href="../../components/navbar">navbar</a>. It aligns to the bottom of the navigation item and stretches to its full width.
-    code: '@include bb-border(nav)'
-    example: '<div class="bb-subsection-heading" style="border-bottom: 4px solid #1c84c6;">Navigation selection underline</div>'
-    styling: '4px solid #1c84c6'
+<h1 class="bb-page-heading">Line styles</h1>
 
-  - usage: This line style separates <a href="../../components/tabset">tab</a> controls from their contents. It aligns to the bottom of the tab controls and stretches to the full width of the container.
-    code: '@include bb-border(tab)'
-    example: '<div class="bb-subsection-heading" style="border-bottom: 2px solid #1c84c6;">Tab underline</div>'
-    styling: '2px solid #1c84c6'
+<p>SKY UX uses various line styles to bound, divide, and highlight content. The lines are designed to maintain visual hierarchy regardless of how complex pages become or how many nested containers they include.</p>
+<p>As with other types of formatting or decoration in SKY UX, use lines minimally. Remember that white space and positioning can group or highlight content without extra subdivisions or boundaries.</p>
+<p>SKY UX uses 3 generalized line styles that you can apply with a SASS mixin.</p>
 
-  - usage: This line affixes to the top of <a href="../../components/tiles">tiles</a>. It extends above the top of the tile and stretches to its full width.
-    code: '@include bb-border(tile)'
-    example: '<div class="bb-subsection-heading" style="border-bottom: 4px solid #e7eaec;">Tile crown</div>'
-    styling: '4px solid #e7eaec'
+<h2 class="bb-section-heading" id="generic-lines">Line styles</h2>
+  
+<div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="bb-subsection-heading" style="border-bottom: 1px solid #e2e3e4;">Light separator</div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>The light separator is a very light gray, single-pixel line that subdivides content within a container. Do not use it at the edge of a container. Only use it when it is adjacent to white space on both sides.</div>
+        <p>SKY UX uses this style for:</p>
```

Comment:
This line is missing space between it and the preceding paragraph. I think it would look better to just pull this text into the previous paragraph. If you prefer to keep it as a separate paragraph, I think we should add `p` tags around the preceding paragraph to give it the necessary spacing.